lib/agenda/database.js

```diff
@@ -25,8 +25,13 @@ module.exports = function(url, collection, options, cb) {
     url = 'mongodb://' + url;
   }
 
+  let reconnectOptions = {autoReconnect: true, reconnectTries: Number.MAX_SAFE_INTEGER, reconnectInterval: this._processEvery};
+  if (options && typeof options.useUnifiedTopology !== 'undefined' && options.useUnifiedTopology === true) {
```

Comment:
I think we can improve this to a falsy check to also accommodate `null`, `undefined` and `false`.

```suggestion
  if (options && options.useUnifiedTopology && options.useUnifiedTopology === true) {
```

What do you think?